README.md

```diff
@@ -47,6 +49,7 @@
 - 10.2.2. Varför används klasser och arv i C#?
 - 10.2.3. Varför använder vi public class i C#?
 - 10.2.4. Vad använder vi private fileds till i C#?
+- 10.2.5. Vad är factory pattern?
```

Comment:
Could be covered by goal 10.1.3. Not sure we need a specific question, students get to choose two patterns and explain them.